recipes/portaudio/all/conanfile.py

```diff
@@ -56,110 +51,32 @@ def system_requirements(self):
-                env_args = []
-                if self.options.cpp_bindings:
-                    env_args.append("-j1")
-                env.make(args=env_args)
-            else:
-                env.make(target = "lib/libportaudio.la")
-            if self.settings.os == "Macos" and self.options.shared:
-                self.run('cd lib/.libs && for filename in *.dylib; do install_name_tool -id $filename $filename; done')
-        else:
-            cmake = CMake(self)
-            cmake.definitions["MSVS"] = self.settings.compiler == "Visual Studio"
-            cmake.definitions["PA_BUILD_STATIC"] = not self.options.shared
-            cmake.definitions["PA_BUILD_SHARED"] = self.options.shared
-            cmake.configure()
-            cmake.build()
+        cmake = self._configure_cmake()
+        cmake.build()
 
     def package(self):
-        self.copy("FindPortaudio.cmake", ".", ".")
-        self.copy("*.h", dst="include", src=os.path.join(self.sources_folder, "include"))
-        if self.options.cpp_bindings:
-            self.copy("*.hxx", dst="include/portaudiocpp", src=os.path.join(self.sources_folder, "bindings/cpp/include/portaudiocpp"))
-        self.copy(pattern="LICENSE*", dst="licenses", src=self.sources_folder,  ignore_case=True, keep_path=False)
-
-        if self.settings.os == "Windows":
-            if self.settings.compiler == "Visual Studio":
-                self.copy(pattern="*.lib", dst="lib", keep_path=False)
-                if self.options.shared:
-                    self.copy(pattern="*.dll", dst="bin", keep_path=False)
-                self.copy(pattern="*.pdb", dst="bin", keep_path=False)
-            else:
-                if self.options.shared:
-                    self.copy(pattern="*.dll.a", dst="lib", keep_path=False)
-                    self.copy(pattern="*.dll", dst="bin", keep_path=False)
-                else:
-                    self.copy(pattern="*static.a", dst="lib", keep_path=False)
-
-        else:
-            if self.options.shared:
-                if self.settings.os == "Macos":
-                    self.copy(pattern="*.dylib", dst="lib", src=os.path.join( "lib", ".libs"))
-                else:
-                    self.copy(pattern="*.so*", dst="lib", src=os.path.join( "lib", ".libs"))
-            else:
-                self.copy("*.a", dst="lib", src=os.path.join("lib", ".libs"))
-
-        if self.options.cpp_bindings:
-            if self.options.shared:
-                if self.settings.os == "Macos":
-                    self.copy(pattern="*.dylib", dst="lib", src=os.path.join("bindings/cpp/lib", ".libs"))
-                else:
-                    self.copy(pattern="*.so*", dst="lib", src=os.path.join("bindings/cpp/lib", ".libs"))
-            else:
-                self.copy(pattern="*.a", dst="lib", src=os.path.join("bindings/cpp/lib", ".libs"))
-
-
+        cmake = self._configure_cmake()
+        cmake.install()
```

Comment:
Similar question, just to make sure: CMake is now installing all relevant files on all platforms?